2023/did-wg.html

```diff
@@ -182,10 +179,7 @@ <h2>Scope</h2>
         </ul>
 
         <p>
-The Working Group may also publish new Working Group Notes, Editors Drafts, or
-Working Drafts for DID Resolution or DID URL Dereferencing, or of DID Method
-Specifications. Any new documents intended for Recommendation will be
-limited to the Working Draft maturity level.
+The Working Group may also publish new Working Group Notes.
```

Comment:
```suggestion
```

Remove the Notes.  The working group will not pulish any new notes.  See also change in "Out of Scope" section.

Reply:
I cannot suggest changes in the "Out of Scope" section, because this is not my pull request.  @pchampin , please implement @iherman 's suggestion to state that Notes are out of scope.

Reply:
Making all possible Notes out of scope for the next DID WG would be a severe limitation. I'm not understanding the reasoning behind this?
The nature of a Working Group is to produce Recommendations. Sometimes these Recommendations are benefitted by the additional publication of useful documents that are associated with the Recommendation.
A ban on all Notes would prohibit the next DID WG from publishing anything other than Recommendation track documents. It would formally put out of scope any updates to the DID Method Rubric, DID Use Cases, DID Implementation Guide, etc. 
Without a strong argument from a large group in support of removing Notes from the charter, I could not support this step.

Reply:
I agree with @brentzundel. I can't think of a recent W3C WG that was limited in this way from publishing Notes. Digital Bazaar would not support such a limitation because part of what a maintenance WG needs to do is maintain the Notes it's already published... the Use Cases, Rubric, and Implementation Guide being at least among the set of Notes that need to be maintained.

Reply:
@brentzundel 

>  It would formally put out of scope any updates to the DID Method Rubric, DID Use Cases, DID Implementation Guide, etc.

This is a straw man argument.  I have not proposed a ban on all notes.  I have proposed a ban on new notes.  If there are new notes that you want, let us know now.

@msporny 

You are supporting a straw man argument.  If there are new notes that you want, let us know now.

Reply:
Unfortunately, W3C Process would consider any update to a previously published Note as a new Note. That's how it works. A ban on new Notes is a ban on all Notes.

This is not a straw man. We don't have some dark hidden agenda. We are simply pointing out some consequences of the changes you are insisting on. 

If new Notes are out of scope for the working group, then it will publish no Notes.

Reply:
@brentzundel 
> If new Notes are out of scope for the working group, then it will publish no Notes.

In the context of this discussion, @iherman has [already advised](https://github.com/w3c/charter-drafts/pull/448#discussion_r1371934455) listing the Notes to be updated explicitly.  It's completely valid to say that unforseen Notes will not be published.

Reply:
While it may be valid to say that no unforeseen Notes will be published, I am failing to understand why such a thing would be desirable. 
Manu and I have expressed concerns that a prohibition against all unforeseen Notes (even if an exception is carved out for updates to existing Notes) could potentially cause significant difficulties for the working group. I am not understanding what the benefit of such a prohibition would be, either for the working group or for the progression of DIDs in general. 

@rxgrant Could you please outline your reasons for wanting to prohibit the working group from publishing Notes?

Reply:
DCD does not approve this charter in [its current form](https://github.com/w3c/charter-drafts/pull/448/commits/d8df0476a32f2ea4331c4f589f4593cf6bcba468).

I would not be opposed to listing a DID Resolution Implementation Guide as a foreseen Note, if the consensus process for editing that Note is written down beforehand and is a major advance from the process used for the DID Implementation Guide, which I have described problems with [here](https://github.com/w3c/charter-drafts/pull/448#discussion_r1381888017).

I would also be in favor of wording that **allows** unforseen Notes _related to DID Resolution_ when that wording allows **no wiggle room** for Notes about DID Methods.

The following must be explicitly prohibited in the Out of Scope section:
- work on particular DID Methods
- publishing DID Methods as Notes

Until I see better wording, I think the best way to achieve this is to hold the line on no unforseen Notes.

Reply:
I have no problem with the prohibition against DID Methods being extended to Notes. Such a thing seems unnecessary to me, considering that publishing something as a Note that would ever be considered for the Recommendation track is already out of bounds. 
Extending that prohibition against Notes to include all possible unforseen Notes is not something I could support.

Reply:
Hmm since DID methods were removed in https://github.com/w3c/charter-drafts/pull/448/commits/aacf57d4815121e3946c0edef0c3ad8d894d0547, would it "hurt" to explicitly mention DID methods as out-of-scope? But not prohibit other Notes? It seems this would address the concerns?

Reply:
> Such a thing seems unnecessary to me, considering that publishing something as a Note that would ever be considered for the Recommendation track is already out of bounds.

I generally agree with giving the ability for the WG to publish additional notes is useful.

However, I find the characterization that consensus doesn't apply to Notes to be out of order.

Consensus applies to everything the WG does.

Since that seems to be in doubt, I recommend we add that notion explicitly.

Reply:
@rxgrant, my understanding is that [@jandrieu's proposed change](https://github.com/w3c/charter-drafts/pull/448#discussion_r1385594681) adequately addresses your issue about notes (that's how I read [your response](https://github.com/w3c/charter-drafts/pull/448#discussion_r1390086133)). Therefore, I'm resolving this conversation.

Reply:
unresolving this conversation, as [I had misunderstood](https://github.com/w3c/charter-drafts/pull/448#issuecomment-1816289464) @rxgrant's comment. My apologies.

Reply:
```suggestion
The Working Group may also publish new Working Group Notes. Working Group Notes will not include DID method specifications. The Working Group however welcomes and may support efforts to standardize DID Methods in other Working Groups and/or organizations.
```

@rxgrant and others, how about something like this as an idea?

Reply:
> The Working Group however welcomes and may support efforts to standardize DID methods in other Working Groups and/or organizations.

It's not okay to me to ask the WG to "support efforts" elsewhere.  That has all the same problems as doing something in the WG!

Reply:
proposal: Notes should not include submissions for DID Methods and should not address issues regarding any particular DID Method.  The WG welcomes DID Method standardization outside the WG (whether in the W3C or in other standards bodies).